.travis.yml

```diff
@@ -4,34 +4,18 @@ language: python
 
 matrix:
   include:
-    - python: 2.7
-      env: TOXENV=py27-django111
     - python: 3.5
-      env: TOXENV=py35-django111
-    - python: 3.5
-      env: TOXENV=py35-django111
-    - python: 3.5
-      env: TOXENV=py35-django20
-    - python: 3.5
-      env: TOXENV=py35-django21
-    - python: 3.5
-      env: TOXENV=py35-django22
+      env: TOXENV=django22
     - python: 3.5
       env: TOXENV=quality
     - python: 3.5
       env: TOXENV=docs
-    - python: 3.6
-      env: TOXENV=py36-django111
-    - python: 3.6
-      env: TOXENV=py36-django20
-    - python: 3.6
-      env: TOXENV=py36-django21
-    - python: 3.6
-      env: TOXENV=py36-django22
+    - python: 3.8
+      env: TOXENV=django22
```

Comment:
we should also run quality and docs with Python 3.8.

Reply:
Removed matrix to run test for both python versions.